# NON GRADED CHALLENGE 5

Apakah $\begin{bmatrix}1\\3\end{bmatrix}$ adalah eigenvector dari $\begin{bmatrix}1 &-1\\ 6&-4\end{bmatrix}$? Jika iya, berapakah eigenvalue-nya?

Jawaban:

In [5]:
from numpy import array,linalg
A = array([[1,-1], [6,-4]])
A


array([[ 1, -1],
       [ 6, -4]])

In [6]:
from numpy.linalg import eig
eigval, eigvec = eig(A)

In [7]:
eigval

array([-1., -2.])

In [8]:
eigvec

array([[0.4472136 , 0.31622777],
       [0.89442719, 0.9486833 ]])

Jadi $\begin{bmatrix}1\\3\end{bmatrix}$ bukan eigenvector dari $\begin{bmatrix}1 &-1\\ 6&-4\end{bmatrix}$

eigenvector yang benar adalah $\begin{bmatrix}0.4 & 0.3\\0.89 & 0.9\end{bmatrix}$ dengan eigenvaluenya $\begin{bmatrix}-1 \\ -2\end{bmatrix}$